### DISCUSSION TITLE
Brackets query

### DISCUSSION THREAD

**Discussion by dbeaver12** [dbeaver/dbeaver#19793] [General]
### Description

Hi..
With ref to a software research, we would like to check on these prior to use in Manchester Metropolitan university one of our departments-

1. When was software last updated
2. Is software still on active development
3. Are the software provider committed to patching the software in event of security incident or known security vulnerabilities.

Please could answer the above-mentioned question about the Brackets software.

Thanks, and Regards,
Jesu
Jesu Mathew 
Software Licensing Administrator

### DBeaver Version

nil

### Operating System

nil

**Comment by LonwoLonwo** [comment]
Hello.

1. Our last (23.0.3) release was today.
You can see our release calendar here: https://github.com/dbeaver/dbeaver/milestones
2. Yes, it is. We have releases every two weeks usually.
3. If we find a vulnerability, we fix this place with vulnerability.

If any questions left, feel free to ask.
